src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -171,15 +171,35 @@
                 <h3>Request Headers</h3>
                 {{ include('@WebProfiler/Profiler/bag.html.twig', { bag: collector.requestheaders, labels: ['Header', 'Value'], maxDepth: 1 }, with_context = false) }}
 
-                <h3>Request Content</h3>
+                <h3>
```

Comment:
Should be reverted to prevent merge conflicts

Reply:
👌